content/departments/cloud/technical-docs/operations.md

```diff
@@ -358,9 +359,12 @@ Once you have identified a repo is constantly failing to be updated/fetched, exe
 ```sh
 mg ssh
 docker exec -it gitserver-0 sh
-cd /data/repos/<repo_name>
-git prune && git fetch
-# look for errors, no output indicates clean repo
+cd /data/repos/<repo_name>/.git
+cat sgm.log
+cat gc.log
+# look for errors and numbers of failures
+# Also run
+git prune && git fetch # check for errors
```

Comment:
@daxmc99 the issue I see here is that only cloud team can do it, right? b/c other engineers cannot ssh to instance?

Reply:
We may provide temporary access in the future to allow other engineers to debug issues (at least this is my understanding)

I agree we shouldn't need ssh access though

Reply:
Can we have a GraphQL mutation to inspect the status / remove the offending repo?